devel/ansible/roles/bodhi/tasks/main.yml

```diff
@@ -100,12 +97,25 @@
   args:
       creates: /home/vagrant/.db-created
 
+# This isn't packaged in Fedora yet, but it's only a development tool (we should still add it)
+- name: pip install pyramid
```

Comment:
can you explain the reason about moving from koji provided packages to pip provided packages?

Reply:
the reason is i'm a silly goose.

Was playing around with trying to test out pyramid2, and left this lying around. removed it now. thanks!